sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/MapCameraManagerDelegate.kt

```diff
@@ -28,9 +28,9 @@ interface MapCameraManagerDelegate {
    */
   fun cameraForCoordinateBounds(
     bounds: CoordinateBounds,
-    padding: EdgeInsets,
-    bearing: Double?,
-    pitch: Double?
+    padding: EdgeInsets = EdgeInsets(0.0, 0.0,0.0, 0.0),
```

Comment:
Do we want add tests here? Verifying that correct default values are picked.

Reply:
To me that feels like we are testing the default parameter feature of kotlin but for your sanity it will add it :)